services/graph/pkg/config/config.go

```diff
@@ -62,6 +62,8 @@ type LDAP struct {
 	UserNameAttribute        string `yaml:"user_name_attribute" env:"LDAP_USER_SCHEMA_USERNAME;GRAPH_LDAP_USER_NAME_ATTRIBUTE" desc:"LDAP Attribute to use for username of users."`
 	UserIDAttribute          string `yaml:"user_id_attribute" env:"LDAP_USER_SCHEMA_ID;GRAPH_LDAP_USER_UID_ATTRIBUTE" desc:"LDAP Attribute to use as the unique ID for users. This should be a stable globally unique ID like a UUID."`
 	UserEnabledAttribute     string `yaml:"user_enabled_attribute" env:"LDAP_USER_ENABLED_ATTRIBUTE;GRAPH_USER_ENABLED_ATTRIBUTE" desc:"LDAP Attribute to use as a flag telling if the user is enabled or disabled."`
+	LocalUserDisable         bool   `yaml:"local_user_disable" env:"LDAP_LOCAL_USER_DISABLE;GRAPH_LOCAL_USER_DISABLE" desc:"Boolean to control behavior for disabling users. If true, disabling a user via API will add the user to a group for disabled users."`
```

Comment:
```suggestion
	LocalUserDisable         bool   `yaml:"local_user_disable" env:"LDAP_LOCAL_USER_DISABLE;GRAPH_LOCAL_USER_DISABLE" desc:"A boolean value to control the behavior for disabling users. If set to 'true', disabling a user via API will add the user to the configured group for disabled users, no action otherwise. Supported values are 'true' and 'false'."`
```